tlspuffin/src/wolfssl/mod.rs

```diff
@@ -263,9 +273,9 @@ impl Put<TLSProtocolBehavior> for WolfSSL {
         unsafe { version().to_string() }
     }
 
-    fn set_deterministic(&mut self) -> Result<(), puffin::error::Error> {
```

Comment:
Yes. I noticed the change and we discussed it offline. I will rebase my PR while taking this into account. Not a big deal since we don't provide an actual implementation of the C PUT, for now...